Source/Core/Core/HW/WiimoteCommon/DataReport.cpp

```diff
@@ -195,26 +191,28 @@ struct ReportExt21 : NoCore, NoAccel, NoIR, IncludeExt<0, 21>
 struct ReportInterleave1 : IncludeCore, IncludeIR<3, 18, 0>, NoExt
 {
   // FYI: Only 8-bits of precision in this report, and no Y axis.
-  // Only contains 4 MSB of Z axis.
-
   void GetAccelData(AccelData* accel) const override
   {
-    accel->x = data_ptr[2] << 2;
+    // Y axis only has 8 bits of precision. (converted to 10)
```

Comment:
Thanks, it's a typo in the comment.

Wii Remote accelerometer component X usually has 10 bits of precision, Y and Z usually have 9.
All 3 are converted to 10 for consistent handling elsewhere.

The "interleaved" reports, (https://wiibrew.org/wiki/Wiimote#0x3e_.2F_0x3f:_Interleaved_Core_Buttons_and_Accelerometer_with_36_IR_bytes) which are literally never used but implemented because the hardware supports it, have only 8 bits of precision for each component, converted to 10 again for code elsewhere to not be special-cased.